src/dc_receive_imf.rs

```diff
@@ -2892,6 +2902,65 @@ mod tests {
         assert_eq!(msgs.len(), 2);
     }
 
+    #[async_std::test]
+    async fn test_majordomo_mailing_list() {
+        let t = TestContext::new_alice().await;
+        t.ctx
+            .set_config(Config::ShowEmails, Some("2"))
```

Comment:
Note that you can now use `t` instead of `t.ctx` (also below)

Reply:
in theory, i know that :) i will do a commit and change that.